### ISSUE TITLE
[Accessibility] Checkbox: Cannot Toggle in Scan Mode

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#5605]
### Bug Report 
 
- __Package version(s)__: Latest, on https://developer.microsoft.com/en-us/fabric
- __Browser and OS versions__: Edge/Narrator, on Windows 10 Enterprise; Version 1803; OS 17134.165
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: **High**
 
Products/sites affected: All
 
#### Describe the issue:
 
Users cannot toggle Checkboxes in Scan Mode on Narrator.
- Turn on Scan Mode in Narrator.
- Go to https://developer.microsoft.com/en-us/fabric#/components/checkbox
- Using the arrow keys, in Scan Mode, navigate to the Checkboxes in the examples.
- Hit 'Space' when focus is on the Checkbox text.

#### Actual behavior:
 
Hitting 'Space' activates the last actionable element before the Checkbox.
 
#### Expected behavior:
 
Hitting 'Space' toggles the Checkbox.
 
### If applicable, please provide a codepen repro:

Can repro here: https://developer.microsoft.com/en-us/fabric#/components/checkbox

Expected behaviour should match: https://www.w3.org/TR/wai-aria-practices/examples/checkbox/checkbox-1/checkbox-1.html

**Comment by betrue-final-final** [comment]
This is a real bug. Yes, the behavior should match https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_checked

**Comment by JasonGore** [comment]
I started looking at this and I'm not getting the same behavior on this [CodePen](https://codepen.io/jasongore/pen/bjQpLg) as the site exhibits. I'm not sure what the difference is yet.

**Comment by micahgodbolt** [comment]
I'm not able to repro this in my local or on our public site.

Enable Scan mode
arrow to the checkbox
hit space
checkbox becomes checked.

If you can still repro this issue (it might have been recently fixed) feel free to open another issue with detailed repro instructions.

**Comment by HalcyonPride** [comment]
Narrator team confirmed this is a Bug on their end that will be fixed in their next release.
